<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的笔记</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>258</width>
         <height>613</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="searchWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="treeViewModel" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>807</width>
         <height>613</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>加粗</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>倾斜</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>下划线</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>左对齐</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>右对齐</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="text">
              <string>居中对齐</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_9">
             <property name="text">
              <string>图片</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>视频</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>笔记</string>
    </property>
    <addaction name="action_new_node"/>
    <addaction name="actionnew_dir"/>
    <addaction name="actiondel_node"/>
    <addaction name="actiondel_dir"/>
    <addaction name="actionquit_node"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>账号</string>
    </property>
    <addaction name="actionswitch_account_2"/>
    <addaction name="actionbase_info"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionabout"/>
    <addaction name="actionupdate"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_new_node">
   <property name="icon">
    <iconset resource="menuicon.qrc">
     <normaloff>:/icon/new.png</normaloff>:/icon/new.png</iconset>
   </property>
   <property name="text">
    <string>新建笔记</string>
   </property>
  </action>
  <action name="actionnew_dir">
   <property name="icon">
    <iconset resource="menuicon.qrc">
     <normaloff>:/icon/newDir.png</normaloff>:/icon/newDir.png</iconset>
   </property>
   <property name="text">
    <string>新建目录</string>
   </property>
  </action>
  <action name="actiondel_node">
   <property name="icon">
    <iconset resource="menuicon.qrc">
     <normaloff>:/icon/delFile.png</normaloff>:/icon/delFile.png</iconset>
   </property>
   <property name="text">
    <string>删除笔记</string>
   </property>
  </action>
  <action name="actiondel_dir">
   <property name="icon">
    <iconset resource="menuicon.qrc">
     <normaloff>:/icon/del.png</normaloff>:/icon/del.png</iconset>
   </property>
   <property name="text">
    <string>删除目录</string>
   </property>
  </action>
  <action name="actionquit_node">
   <property name="icon">
    <iconset resource="menuicon.qrc">
     <normaloff>:/icon/exit.png</normaloff>:/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出应用</string>
   </property>
  </action>
  <action name="actionswitch_account">
   <property name="text">
    <string>switch account</string>
   </property>
  </action>
  <action name="actionswitch_account_2">
   <property name="icon">
    <iconset resource="menuicon.qrc">
     <normaloff>:/icon/qiehuan.png</normaloff>:/icon/qiehuan.png</iconset>
   </property>
   <property name="text">
    <string>切换账号</string>
   </property>
  </action>
  <action name="actionbase_info">
   <property name="icon">
    <iconset resource="menuicon.qrc">
     <normaloff>:/icon/ziliao.png</normaloff>:/icon/ziliao.png</iconset>
   </property>
   <property name="text">
    <string>基本信息</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="icon">
    <iconset resource="menuicon.qrc">
     <normaloff>:/icon/about.png</normaloff>:/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionupdate">
   <property name="icon">
    <iconset resource="menuicon.qrc">
     <normaloff>:/icon/shengji.png</normaloff>:/icon/shengji.png</iconset>
   </property>
   <property name="text">
    <string>升级</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>searchWidget</class>
   <extends>QWidget</extends>
   <header location="global">searchwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>treeViewModel</class>
   <extends>QWidget</extends>
   <header location="global">treeviewmodel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="menuicon.qrc"/>
 </resources>
 <connections/>
</ui>
